lte/gateway/python/magma/pipelined/bridge_util.py

```diff
@@ -96,22 +96,6 @@ def get_ofport(interface_name):
             )
         return int(port_num)
 
-    @staticmethod
-    def get_mac_address(interface_name):
```

Comment:
There's another function `get_virtual_iface_mac` in `utils.py` that you can probably delete.